src/library_fs.js

```diff
@@ -838,7 +838,7 @@ mergeInto(LibraryManager.library, {
       if (!link.node_ops.readlink) {
         throw new FS.ErrnoError(ERRNO_CODES.EINVAL);
       }
-      return link.node_ops.readlink(link);
+      return PATH.resolve('.', link.node_ops.readlink(link));
```

Comment:
Need this to normalize paths that come out of non-Node filesystems. Because from nodefs we now call `PATH.join` the path is normalized. We need to do this for all paths to get consistent results in the test output between node/non-node environment.